Wikipedia/Code/AppDelegate.m

```diff
@@ -99,6 +99,7 @@ - (void)applicationWillEnterForeground:(UIApplication *)application {
 - (void)applicationDidBecomeActive:(UIApplication *)application {
     // Restart any tasks that were paused (or not yet started) while the application was inactive. If the application was previously in the background, optionally refresh the user interface.
     [[NSUserDefaults wmf] wmf_setAppBecomeActiveDate:[NSDate date]];
+    NSUserDefaults.wmf.shouldShowLastReadArticleOnResume = [self shouldShowLastReadArticleOnResume];
```

Comment:
The app should only need to show last article on the resume on a full quit & relaunch - `didBecomeActive` is called on every re-launch of the app even if it wasn't fully terminated. This should probably move to `didFinishLaunching`